test/intl402/Temporal/Calendar/prototype/dateFromFields/one-of-era-erayear-undefined.js

```diff
@@ -3,17 +3,17 @@
 
 /*---
 esid: sec-temporal.calendar.prototype.datefromfields
-description: Throw a RangeError if only one of era/eraYear fields is present
+description: Throw a TypeError if only one of era/eraYear fields is present
 features: [Temporal]
 ---*/
 
 const base = { year: 2000, month: 5, day: 2, era: 'ce' };
 const instance = new Temporal.Calendar('gregory');
-assert.throws(RangeError, () => {
+assert.throws(TypeError, () => {
   instance.dateFromFields({ ...base });
 });
 
 const base2 = { year: 2000, month: 5, day: 2, eraYear: 1 };
-assert.throws(RangeError, () => {
+assert.throws(TypeError, () => {
```

Comment:
This (and the similar change in yearMonthFromFields) is an additional pre-existing bug that I found while working on this. CalendarResolveFields says "The operation throws a **TypeError** exception if the properties of _fields_ are internally inconsistent within the calendar or insufficient to identify a unique instance of _type_ in the calendar," but we were expecting a RangeError here.